src/PointwiseFunctions/GeneralRelativity/Christoffel.hpp

```diff
@@ -0,0 +1,22 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+///\file
+/// Defines Functions to calculate Christoffel symbols
+
+#pragma once
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief Computes Christoffel symbol of first kind from derivative of metric
+ *
+ * \details Computes Christoffel symbol \f$\Gamma_{abc}\f$ as:
+ * \f$ \Gamma_{abc} = 1/2 ( \partial_a g_{bc} + \partial_b g_{ac} - \partial_a
+ * g_{bc}) \f$
+ *  where \f$g_{bc}\f$ is either a spatial or spacetime metric
```

Comment:
Extra whitespace at beginning of line

Reply:
I expressed myself poorly, but what I meant to say here was: there is one extra space before "where" that should be removed.